<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>promptForm</class>
 <widget class="QWidget" name="promptForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puzzle stacker</string>
  </property>
  <widget class="QPushButton" name="open_shuffle_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rozsypane puzzle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Powrót</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_solved_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ułożone puzzle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Dalej</string>
   </property>
  </widget>
  <widget class="QLabel" name="shuffle_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>315</width>
     <height>215</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/newPrefix/logo.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="solved_image">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>315</width>
     <height>215</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>315</width>
     <height>215</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/logo.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="prompt_image.qrc"/>
 </resources>
 <connections/>
</ui>
